src/wp-includes/media.php

```diff
@@ -1903,17 +1903,49 @@ function wp_filter_content_tags( $content, $context = null ) {
 	return $content;
 }
 
+/**
+ * Adds a fallback mechanism to replace webp images with alternative mime types on older browsers.
+ *
+ * @since 6.1.0
+ *
+ * @global bool $_wp_image_mime_fallback_should_load
+ */
+function wp_print_image_mime_fallback_script() {
+	global $_wp_image_mime_fallback_should_load;
+
+	// Bail early if an image has no additional mime.
+	if ( true !== $_wp_image_mime_fallback_should_load ) {
+		return;
+	}
+
+	$suffix   = wp_scripts_get_suffix();
+	$version  = 'ver=' . get_bloginfo( 'version' );
+	$settings = array(
+		'restApi'           => esc_url_raw( trailingslashit( get_rest_url() ) ),
+		'imagemimefallback' => apply_filters( 'script_loader_src', includes_url( "js/wp-image-mime-fallback$suffix.js?$version" ), 'imagemimefallback' ),
```

Comment:
A few small problems here:
* The comment about the filter is missing.
* The JS field should use camel-case.
* The second filter parameter should be similar to a script handle _if_ this file was registered as a script, i.e. probably similar to the production file name.
```suggestion
		/** This filter is documented in wp-includes/class.wp-scripts.php */
		'imageMimeFallbackScript' => apply_filters( 'script_loader_src', includes_url( "js/wp-image-mime-fallback$suffix.js?$version" ), 'wp-image-mime-fallback' ),
```

Reply:
Thank you for the review and guidance.